device-types/Supermicro/MBD-X8STi.yaml

```diff
@@ -0,0 +1,40 @@
+---
+manufacturer: Supermicro
+model: Motherboard X8STi
+slug: supermicro-mbd-x8sti
+part_number: MBD-X8STi
+u_height: 1
```

Comment:
As this is a motherboard it should have `subdevice_role: child` and as part of that it needs to be u_height: 0